api/controllers/calendarEntry.controller.ts

```diff
@@ -234,32 +255,8 @@ export const updateCalendarEntry = async (
 ) => {
   const { id } = req.params;
   try {
-    const originalEntry = await CalendarEntry.findByIdAndUpdate(
-      id,
-      req.body as CalendarEntry,
-    );
+    const originalEntry = await CalendarEntry.findByIdAndUpdate(id, req.body);
```

Comment:
What is the reason the type is removed here?

Reply:
Oversight, added it back in